.github/workflows/release-clarity-vscode.yaml

```diff
@@ -4,10 +4,8 @@ on:
   push:
     tags:
       - 'v*'
-    paths-ignore:
-      - '**/CHANGELOG.md'
-      - '**/package.json'
-      - '**/package-lock.json'
+    paths:
+      - components/clarity-vscode
```

Comment:
Really cool trick, I'll keep it in mind.
We would need to include a few other paths:
- components/clarity-repl
- components/clarity-lsp
- components/clarinet-files
- components/clarinet-deployments

Reply:
Handled in PR #684